subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/DefaultLenientConfiguration.java

```diff
@@ -267,11 +267,12 @@ public void run(BuildOperationContext context) {
 
             @Override
             public BuildOperationDescriptor.Builder description() {
-                String displayName = "Resolve files of " + configuration.getIdentityPath();
+                String displayName = "Resolve files of " + resolveContext.getIdentityPath();
                 return BuildOperationDescriptor
                     .displayName(displayName)
                     .progressDisplayName(displayName)
-                    .details(new ResolveArtifactsDetails(configuration.getPath()));
+                    // TODO: Can we update this to use the identity path?
+                    .details(new ResolveArtifactsDetails(resolveContext.getProjectPath().toString()));
```

Comment:
Can path be `null` here, as it was above?

Reply:
It seems this is always non-null. Its just the project component ID returned by `Module` that's null when we're executing settings scripts.